src/components/RadioButtonWithLabel.tsx

```diff
@@ -1,84 +1,70 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import {View} from 'react-native';
-import _ from 'underscore';
+import React, {ComponentType} from 'react';
+import {StyleProp, View, ViewStyle} from 'react-native';
 import styles from '@styles/styles';
 import FormHelpMessage from './FormHelpMessage';
 import * as Pressables from './Pressable';
 import RadioButton from './RadioButton';
 import Text from './Text';
 
-const propTypes = {
+type RadioButtonWithLabelProps = {
     /** Whether the radioButton is checked */
-    isChecked: PropTypes.bool.isRequired,
+    isChecked: boolean;
 
     /** Called when the radioButton or label is pressed */
-    onPress: PropTypes.func.isRequired,
+    onPress: () => void;
 
     /** Container styles */
-    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+    style?: StyleProp<ViewStyle>;
 
     /** Text that appears next to check box */
-    label: PropTypes.string,
+    label: string;
```

Comment:
```suggestion
    label?: string;
```

Reply:
Since it is a RadioButtonWithLabel we always need to have a label

Reply:
changed